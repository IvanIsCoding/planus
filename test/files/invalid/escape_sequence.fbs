table Foo {
  x: string = "\xjj";
}