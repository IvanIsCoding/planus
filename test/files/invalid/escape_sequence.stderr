[0m[1m[38;5;9merror[0m[1m: Invalid escape sequence[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/escape_sequence.fbs:2:16
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: string = "[0m[31m\x[0mjj";
  [0m[34mâ”‚[0m                [0m[31m^^[0m

Error: Bailing because of errors
